infrastructure/src/main/java/org/corfudb/infrastructure/CorfuServer.java

```diff
@@ -176,6 +180,8 @@
 
     private static volatile Thread corfuServerThread;
     private static volatile Thread shutdownThread;
+    // Bind to all interfaces only if no address or interface specified by the user.
+    private static boolean bindToAllInterfaces = false;
```

Comment:
This shouldn't be a static var

Reply:
Its accessed only by static methods.
I have to make it a class level variable as it is reused on restarts/resets.

Reply:
Isn't that passed around in the context anyway?